src/ResourceManager/Resources/Commands.Resources/ActiveDirectory/NewAzureADSpCredentialCommand.cs

```diff
@@ -74,30 +75,21 @@ public override void ExecuteCmdlet()
                     ObjectId = ActiveDirectoryClient.GetObjectIdFromSPN(ServicePrincipalName);
                 }
 
-#pragma warning disable 0618
-                if (!string.IsNullOrEmpty(Password))
-#pragma warning restore 0618
-                {
+                if (Password != null && Password.Length > 0) {
```

Comment:
All beginning brackets should be on a new line.  Please apply across this whole file.